Update CONTRIBUTING.md

Ethan-Arrowood opened the pull request:
MongoDB download page defaults to their Atlas project. I'm proposing we specify users download the MongoDB community server directly to diffuse any potential confusion.

Ethan-Arrowood committed:
Update CONTRIBUTING.md

MongoDB download page defaults to their Atlas project. I'm proposing we specify users download the MongoDB community server directly to diffuse any potential confusion.

raisedadead commented:
I am afraid, they also redirect links to their atlas link by default.

Ethan-Arrowood commented:
Oh no they do. well maybe at least specifying the Community Server edition in the text?

raisedadead commented:
I think you should amend your changes to point to this rather:
https://docs.mongodb.com/manual/administration/install-community/

Ethan-Arrowood committed:
Update CONTRIBUTING.md

Change link to documentation instructions on how to download the community edition.
